homeassistant/components/flipr/config_flow.py

```diff
@@ -23,9 +24,9 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
     _username: str | None = None
     _password: str | None = None
     _flipr_id: str | None = None
-    _possible_flipr_ids: list[str] | None = None
+    _possible_flipr_ids: list[str]
 
-    async def async_step_user(self, user_input=None):
+    async def async_step_user(self, user_input=None) -> FlowResult:
```

Comment:
Please type the whole signature when adding type annotations.

Reply:
Corrected via the recent push.